airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/split_secrets/JsonSecretsProcessorTest.java

```diff
@@ -62,6 +62,97 @@
           + "    }\n"
           + "  }");
 
+  private static final JsonNode ONE_OF_WITH_SAME_KEY_IN_SUB_SCHEMAS = Jsons.deserialize(
```

Comment:
```suggestion
  private static final JsonNode ONE_OF_WITH_SAME_KEY_IN_SUB_SCHEMAS = Jsons.deserialize(
```

Generally we prefer to stick json files like this for testing in a resource so it's easier to read and edit compared to the escaped Java string.